src/client/metric.rs

```diff
@@ -107,6 +99,35 @@ pub enum Space {
     EByte
 }
 
+impl Space {
+    fn from_u8(x: u8) -> Option<Self> {
+        match x {
+            0 => Some(Space::Byte),
```

Comment:
Yeah fair enough. Come to think of it, it would just be extra indirection having `0` defined as a constant when we know its a `Space::Byte` already.